konduit-serving-core/src/main/java/ai/konduit/serving/launcher/command/ConfigCommand.java

```diff
@@ -0,0 +1,263 @@
+                                        .build()
+                                )
+                                .build()),
+                        ServingConfig.builder()
+                                .build(),
+                        null
+                ).toJson()
+        );
+    }
+
+    private JsonObject onnx() {
+        return new JsonObject(
+                new InferenceConfiguration(
+                        Arrays.asList(ModelStep.builder()
+                                .inputName("default")
+                                .outputName("default")
+                                .build()),
+                        ServingConfig.builder()
+                                .build(),
+                        null
+                ).toJson()
+        );
+    }
+
+    private JsonObject pmml() {
+        return new JsonObject(
+                new InferenceConfiguration(
+                        Arrays.asList(ModelStep.builder()
+                                .inputName("default")
+                                .outputName("default")
+                                .modelConfig(
+                                        PmmlConfig.builder()
+                                                .build()
+                                )
+                                .build()),
+                        ServingConfig.builder()
+                                .build(),
+                        null
+                ).toJson()
+        );
+    }
+
+    private JsonObject dl4j() {
+        return new JsonObject(
+                new InferenceConfiguration(
+                        Arrays.asList(ModelStep.builder()
+                                .inputName("default")
+                                .outputName("default")
+                                .modelConfig(
+                                        DL4JConfig.builder()
+                                                .build()
+                                )
+                                .build()),
+                        ServingConfig.builder()
+                                .build(),
+                        null
+                ).toJson()
+        );
+    }
+
+    private JsonObject keras() {
+        return new JsonObject(
+                new InferenceConfiguration(
+                        Arrays.asList(ModelStep.builder()
+                                .inputName("default")
+                                .outputName("default")
+                                .modelConfig(
+                                        KerasConfig.builder()
+                                                .build()
+                                )
+                                .build()),
+                        ServingConfig.builder()
+                                .build(),
+                        null
+                ).toJson()
+        );
+    }
+
+    private void printOrSave(String output) {
+        if(outputFile == null) {
+            log.info(output);
```

Comment:
Not sure if it's possible given the system.out to logging redirect, but this might be one of the cases where a System.out.println would be better than a log.info
The reason: Presumably the user will copy/paste this. With log.info, users need to delete the logging prefix part:
i.e., the output will probably look like this:
```
a.k.s.l.c.ConfigCommand - {
a.k.s.l.c.ConfigCommand -   "someField": "someValue"
a.k.s.l.c.ConfigCommand -   "someObject": {
a.k.s.l.c.ConfigCommand -       "otherField": "otherValue"
a.k.s.l.c.ConfigCommand -   }
a.k.s.l.c.ConfigCommand - }
```

Reply:
Yeah took care of that programmatically. Trying to filter out only relevant information here. Still working on it. I realised there's quite a bit of work needed on spitting out useful logs that's only relevant to a user. Kind of like a simplified way of presenting an error and a possible solution if there's a mistake on the user's end. I'm normally thinking of keeping all the logs in a `command.log` which can be useful for reporting errors.

Reply:
hm... so you're suggesting we split into say "user logs" and "debug/system logs"
Basically the same thing but:
* User logs go to console only, info level and higher, _maybe_ with different formatting (no class prefix)?
* System logs go to file, are debug level and higher?

Something like that would be good for both debugging and usability... let's open an issue to discuss further.
And yes, we no doubt need to do a pass on log levels to ensure good UX for users